source/priority_queue.c

```diff
@@ -263,6 +265,16 @@ bool aws_priority_queue_is_valid(const struct aws_priority_queue *const queue) {
     return pred_is_valid && container_is_valid && backpointers_valid;
 }
 
+bool aws_priority_queue_is_wiped(const struct aws_priority_queue *const queue) {
```

Comment:
I think we should figure out what we're doing with #453 before we add any more of these.

Reply:
+1